les/client.go

```diff
@@ -297,7 +297,7 @@ func (s *LightDummyAPI) Mining() bool {
 // APIs returns the collection of RPC services the ethereum package offers.
 // NOTE, some of these services probably need to be moved to somewhere else.
 func (s *LightEthereum) APIs() []rpc.API {
-	apis := ethapi.GetAPIs(s.ApiBackend)
+	apis := ethapi.GetAPIs(s.ApiBackend, nil)
```

Comment:
passing in nil for the `BlockChain` object im sure would lead to some nil pointers elsewhere.

Reply:
might be able to do this here:
```suggestion
	apis := ethapi.GetAPIs(s.ApiBackend, s.BlockChain())
```